Assets/MRTK/Core/Providers/Hands/ArticulatedHandDefinition.cs

```diff
@@ -28,9 +28,57 @@ public ArticulatedHandDefinition(IMixedRealityInputSource source, Handedness han
         private Dictionary<TrackedHandJoint, MixedRealityPose> unityJointPoses = new Dictionary<TrackedHandJoint, MixedRealityPose>();
         private MixedRealityPose currentIndexPose = MixedRealityPose.ZeroIdentity;
 
-        // Pinch distance thresholds
-        private readonly float enterPinchDistance = 0.02f;
-        private readonly float exitPinchDistance = 0.05f;
+        // Minimum distance between the index and the thumb tip required to enter a pinch
+        private readonly float minimumPinchDistance = 0.015f;
+
+        // Maximum distance between the index and thumb tip required to exit the pinch gesture
+        private readonly float maximumPinchDistance = 0.1f;
+
+        // Default enterPinchDistance value
+        private float enterPinchDistance = 0.02f;
+
+        /// <summary>
+        /// The distance between the index finger tip and the thumb tip required to enter the pinch/air tap selection gesture.
+        /// The pinch gesture enter will be registered for all values less than the EnterPinchDistance. The default EnterPinchDistance value is 0.02 and must be between 0.015 and 0.1. 
+        /// </summary>
+        public float EnterPinchDistance 
+        {
+            get => enterPinchDistance;
+            set
+            {
+                if (value >= minimumPinchDistance && value <= maximumPinchDistance)
+                {
+                    enterPinchDistance = value;
+                }
+                else
+                {
+                    Debug.LogWarning("EnterPinchDistance must be be between 0.015 and 0.1, please change Enter Pinch Distance in the Leap Motion Device Manager Profile");
```

Comment:
Might want to log an error instead of warning here.

Reply:
added